php_phongo.c

```diff
@@ -385,11 +385,7 @@ zend_bool phongo_writeconcernerror_init(zval *return_value, bson_t *bson TSRMLS_
 
 		if (!phongo_bson_to_zval(data, len, &intern->info)) {
 			zval_ptr_dtor(&intern->info);
-#if PHP_VERSION_ID >= 70000
 			ZVAL_UNDEF(&intern->info);
-#else
-			intern->info = NULL;
-#endif
```

Comment:
This isn't really related to PHPC-948. I'm just employing the `ZVAL_UNDEF()` compatibility macro created for the patch and cleaning up some old code.